daffodil-cli/README.md

```diff
@@ -38,6 +38,7 @@ For more information about Daffodil, see https://daffodil.apache.org/.
 
 To execute Daffodil on Linux:
 
+<!-- markdownlint-disable commands-show-output -->
 ```bash
 $ ./bin/daffodil [options]
```

Comment:
Should we just remove this leading $ if it's considered bad practice by linters?

Reply:
Sure, we can just remove the leading $.  According to markdownlint, the shell prompt should be there if we show a command's output, but shouldn't be there if we don't show a command's output.  I didn't know what you would think about removing the dollar sign, so I simply added a comment to disable the warning although I did fix all the other warnings.  I won't bother to change the README again, but I expect whoever edits the README in the future will see that comment and be reminded to fix the warning by removing the dollar sign from the three places it appears instead.